modules/ocf_mirrors/manifests/projects/archlinuxcn.pp

```diff
@@ -0,0 +1,17 @@
+class ocf_mirrors::projects::archlinuxcn {
+  file {
+    '/opt/mirrors/project/archlinuxcn':
+      ensure  => directory,
+      source  => 'puppet:///modules/ocf_mirrors/project/archlinuxcn/',
+      owner   => mirrors,
+      group   => mirrors,
+      mode    => '0755',
+      recurse => true;
+  }
+
```

Comment:
@abizer They said they'll add a lastsync timestamp in a few days for us to monitor updates.